src/app/hooks/useBridgeNetworkSendTx.ts

```diff
@@ -0,0 +1,160 @@
+import { useCallback, useEffect, useState } from 'react';
+import { TransactionConfig } from 'web3-core';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  selectLoadingTransaction,
+  selectTransactions,
+} from 'store/global/transactions-store/selectors';
+import {
+  Transaction,
+  TxStatus,
+  TxType,
+} from 'store/global/transactions-store/types';
+import { actions } from 'store/global/transactions-store/slice';
+import { Asset } from 'types/asset';
+import { ContractName } from 'utils/types/contracts';
+import { useAccount } from './useAccount';
+import { Nullable, Chain } from 'types';
+import { gasLimit } from '../../utils/classifiers';
+import { bridgeNetwork } from '../pages/BridgeDepositPage/utils/bridge-network';
+import { getContract } from '../../utils/blockchain/contract-helpers';
+import { BridgeNetworkDictionary } from '../pages/BridgeDepositPage/dictionaries/bridge-network-dictionary';
+
+export interface TransactionOptions {
+  type?: TxType;
+  approveTransactionHash?: Nullable<string>;
+  asset?: Asset;
+  assetAmount?: string;
+  customData?: { [key: string]: any };
+}
+
+export interface SendTxResponse {
+  txHash: string;
+  txData: Nullable<Transaction>;
+  status: TxStatus | any;
+  loading: boolean;
+}
+
+export interface ResetTxResponseInterface extends SendTxResponse {
+  reset: () => void;
+}
+
+export interface SendTxResponseInterface extends ResetTxResponseInterface {
+  send: (
+    args: any[],
+    config?: TransactionConfig,
+    options?: TransactionOptions,
+  ) => void;
+}
```

Comment:
I think these could be deleted and reused from useContractSendTx hook

Reply:
Removed them and imported from useContractSendTx